lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -464,8 +464,8 @@ withJormungandr tr (JormungandrConfig stateDir block0 mPort logSeverity output e
     startBackend (apiPort, baseUrl) = getGenesisBlockArg block0 >>= \case
         Right (block0H, genesisBlockArg) -> do
             let args = genesisBlockArg ++
-                    [ "--config", nodeConfigFile
-                    , "--log-level", C.toLower <$> show logSeverity
+                    [ -- "--config", nodeConfigFile
```

Comment:
`cardano-wallet` should generate a config file with trusted peers, containing IP and ID (public key) pairs

without that, there is no way for jormungandr to sync, and daedalus cant specify its own `--config` file, because jormungandr crashes if you give it 2 `--config` flags

Reply:
Coming with #850

Reply:
wallet should also pass `--rest-listen` to jormungandr (or embedit into `nodeConfigFile`), based on the `--node-port` the wallet receives